.github/workflows/tests.yml

```diff
@@ -284,6 +284,28 @@ jobs:
         run: |
           ./.ci.sh -t
 
+  macos-15:
+    name: macOS 15.0 (Sequoia)
+    # TODO: when actions supports using ${{env}} in job.*.if and not just job.*.steps.*.if, use this:
+    #if: ${{ env.NIGHTLY }}
+    # in the meantime, copy-paste this:
+    # if: ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
```

Comment:
I imagine we probably want to also keep macos-15 from running on PRs? (Or swap out some other OS?)

I think this one cuts into our macOS M1 allocation for runners, so might be best to save it for the nightly runs.

Reply:
Yes, sounds like a good idea.